### PR TITLE
Add maven-docker plugin support

### PR DESCRIPTION
**Pull request by yevr19**
Add maven-docker plugin support

### CODE REVIEW

File: docs/README-DOCKER-PLUGIN.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
+---
+
+```markdown
```
Comment (Sunagatov) [inline docs/README-DOCKER-PLUGIN.md:4]: delete it

File: docs/README-DOCKER-PLUGIN.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+   ```
+Execute the above command to generate your master password.
+
+3. **Repo Password Encryption** 🔄:
+   ```bash
+   mvn --encrypt-password <your_repository_password_here>
+   ```
+   After running the above command, replace the original password in the `settings.xml` with the encrypted one.
+
+4. **Build the Project** 🏗️:
```
Comment (Sunagatov) [inline docs/README-DOCKER-PLUGIN.md:32]: add space above

File: docs/README-DOCKER-PLUGIN.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## Setup Steps 🛠️
+
+1. **Docker Security Configuration** 🛡️:
+   - Update the `settings.xml` file with your Docker-specific credentials:
+     - `username`
+     - `password`
+     - `registry`
+
+2. **Master Password Generation** 🔑:
```
Comment (Sunagatov) [inline docs/README-DOCKER-PLUGIN.md:20]: add space above

File: docs/README-DOCKER-PLUGIN.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+   ```
+   After running the above command, replace the original password in the `settings.xml` with the encrypted one.
+
+4. **Build the Project** 🏗️:
+   ```bash
+   mvn clean install
+   ```
+   Kick off the build process with the above command.
+
+5. **Image Verification** 🖼️:
```
Comment (Sunagatov) [inline docs/README-DOCKER-PLUGIN.md:38]: add space above

File: docs/README-DOCKER-PLUGIN.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+   Once the build process completes, your code modifications should have been successfully populated into a Docker image. Check your Docker images to confirm.
```
Comment (Sunagatov) [inline docs/README-DOCKER-PLUGIN.md:39]: it should be like this:
```
5. **Image Verification** 🖼️:

    Once the build process completes
```

File: docs/README-DOCKER-PLUGIN.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+     - `password`
+     - `registry`
+
+2. **Master Password Generation** 🔑:
+   ```bash
+   mvn --encrypt-master-password <your_password_here>
+   ```
+Execute the above command to generate your master password.
+
+3. **Repo Password Encryption** 🔄:
```
Comment (Sunagatov) [inline docs/README-DOCKER-PLUGIN.md:26]: add space above

File: docs/README-DOCKER-PLUGIN.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+   - Update the `settings.xml` file with your Docker-specific credentials:
+     - `username`
+     - `password`
+     - `registry`
+
+2. **Master Password Generation** 🔑:
+   ```bash
+   mvn --encrypt-master-password <your_password_here>
+   ```
+Execute the above command to generate your master password.
```
Comment (Sunagatov) [inline docs/README-DOCKER-PLUGIN.md:24]: move it to the right with `tab`

**Review by Sunagatov — COMMENTED** [review]
Rewiewed

**Review by vitaliibredun — APPROVED** [review]
Approved

**Review by Reyzis2021 — APPROVED** [review]

**Review by AiratGimaev — APPROVED** [review]
Approved

**Pull request closed without merging**